Modules/posixmodule.c

```diff
@@ -9,18 +9,6 @@
 
 #define PY_SSIZE_T_CLEAN
 
-#include "Python.h"
-// Include <windows.h> before pycore internal headers. FSCTL_GET_REPARSE_POINT
-// is not exported by <windows.h> if the WIN32_LEAN_AND_MEAN macro is defined,
-// whereas pycore_condvar.h defines the WIN32_LEAN_AND_MEAN macro.
-#ifdef MS_WINDOWS
```

Comment:
Is there any reason why we can not just include `<winioctl.h>` to get `FSCTL_GET_REPARSE_POINT`.